.infra/Pulumi.prod.yaml

```diff
@@ -46,6 +46,8 @@ config:
       secure: AAABAKsjQ79BKL/Msk8ERCFq7TeKGQ6NnzRcQrOoUzQ7hFEDNdq4uxOcG3w5M5aODolHDw==
     cioSiteId:
       secure: AAABAACN2IeoykRGHprr1nYvk1snUvT733+1dp8QI7+WchlaErfJJ3X6LLPYe8BthHnpqw==
+    cioWebhookSecret:
+      secure: AAABAHkXsqVvvtA1VZ3lLrNQoV0Kh3FM31l7QxQVVwQvR4CHbLyqbtg2N+MJ5URkaRwYsMMPv+8RNnRcCiC/v33GjNBc4e57x5MpVpQC2VPUyjMornJI53n7rmeQBtsQ
```

Comment:
This is the testing workspace secret, will revert to production/daily.dev workspace once we know it actually works 🙈